### PR TITLE
Revamped patient details page UI

### LINKED ISSUE #2851: Revamp Patient Detail Page
21a

- [ ] the location is having a lot of white space, revamp the patient detail page to a better UI
- [ ] the heading in the tab is sample test history, change it to patient detail

### PR DESCRIPTION
**Pull request by patelaryan7751**
Fixes #2851 

# Bugs Fixed 

## Revamped patient details page UI and changed page name to the patient details page

image

### CODE REVIEW

**Review by cp-Coder — APPROVED** [review]
LGTM

File: src/Components/Patient/PatientHome.tsx [hunk 1]
```diff
@@ -1088,7 +1089,184 @@ export const PatientHome = (props: any) => {
...
+                    </span>
+                  </div>
+
+                  <div>
+                    <RoleButton
+                      className="btn btn-primary w-full text-xs"
+                      disabled={
+                        !patientData.is_active ||
+                        !(
+                          patientData?.last_consultation?.facility == facilityId
```
Comment (sainak) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                          patientData?.last_consultation?.facility === facilityId
```
https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=9682972

**Comment by nihal467** [comment]
@patelaryan7751 test failed



It is not patient details , it is sample test history

**Comment by patelaryan7751** [comment]
@nihal467  I have changed the title to Sample Test History
image
I also changed the title "Covid Suspect Detail" to "Patient Detail"
image

**Comment by nihal467** [comment]
@patelaryan7751 



- [ ] add a white box, in that mention under-sample test history that there is no data available error message

7a

- [ ] align the badges to the top contents

**Comment by patelaryan7751** [comment]
Hey @nihal467 I have made the above said changes
### Added a white box for both sample details and consultation details
image

### Aligned all badges in patient card
Screenshot from 2022-07-06 11-06-31

**Comment by nihal467** [comment]
@patelaryan7751 


Add District too, because in Manipur, the local body doesn't mention the district name, as we have in Kerala

**Comment by patelaryan7751** [comment]
Hey, @nihal467 added district in location card
image

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**